app/src/main/java/social/entourage/android/onboarding/OnboardingEmailPwdFragment.kt

```diff
@@ -93,7 +92,7 @@ class OnboardingEmailPwdFragment : Fragment() {
     //**********//**********//**********
 
     private fun updateButtonNext(isFromEmail:Boolean) {
-        if (ui_onboard_email_pwd_et_mail?.text!= null && ui_onboard_email_pwd_et_mail.text.toString().isValidEmail()) {
+        if (ui_onboard_email_pwd_et_mail.text.toString().isValidEmail()) {
```

Comment:
ui_onboard_email_pwd_et_mail can be null, can't it ?

Reply:
Same situation as previous comment l.43